source/features/add-pull-request-hotkey.js

```diff
@@ -0,0 +1,16 @@
+import select from 'select-dom';
+
+export default function () {
+	const tabs = select.all('.tabnav-pr .tabnav-tab');
+	const selectedIndex = tabs.indexOf(select('.tabnav-pr .selected'));
+
+	for (const [index, tab] of tabs.entries()) {
+		const keys = [`p ${index + 1}`];
+		if (index === selectedIndex - 1) {
+			keys.push('p ArrowLeft');
+		} else if (index === selectedIndex + 1) {
+			keys.push('p ArrowRight');
+		}
+		tab.setAttribute('data-hotkey', keys);
```

Comment:
Use `.dataset`

Reply:
Good idea. Also... https://github.com/github/eslint-plugin-github/blob/master/docs/rules/no-dataset.md

Reply:
I don't think greppable should be a priority, and if you consistently use `dataset`, you can just do the conversion yourself when grepping.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/166